doc/faq.rst

```diff
@@ -0,0 +1,26 @@
+.. _faq:
+
+Frequently Asked Questions
+==========================
+
+The following are some common questions and issues that arise when deploying
+your own BinderHub.
+
+How can I increase my GitHub API limit?
```

Comment:
Not sure that this is really needed in the FAQ. If included, it should be updated to match the info in CONTRIBUTING.md

Reply:
Where would you recommend putting this info if not in the faq? The other thought I'd had was adding an `Extending your BinderHub` similar to what we have with z2jh, where we could put info like adding a github API token.

Reply:
I would just leave it in the CONTRIBUTING.md unless folks have been asking about it in which case the FAQ is fine.

Reply:
Don't you imagine they'd end up hitting the API limit pretty quickly on their own deployment, regardless of whether they're testing locally? It seems to me like the majority of BinderHub deployments would need to be linked against an API key, since 60 hits an hour isn't much

Reply:
Perhaps in that case, the default should be to recommend the token as a default part of the BinderHub deployment when supporting GitHub (or GitLab...).

Reply:
that's a good idea - lemme put something into a commit and see what you think about that

Reply:
WDYT about latest commit?